Added info in the ReadME file (#4)
TUSAIL/RSE01_Vreede_CR: README.md

# RSE01_Vreede_CR

> This repository has a conversion.py file which converts the temperature into Celsius and Farenheit.

Reviewer: ```suggestion
This repository has functionality that converts temperatures between Kelvin, Celsius and Fahrenheit.
```
